yt_dlp/extractor/firsttv.py

```diff
@@ -129,3 +130,42 @@ def _real_extract(self, url):
         return self.playlist_result(
             self._entries(items), display_id, self._og_search_title(webpage, default=None),
             thumbnail=self._og_search_thumbnail(webpage, default=None))
+
+
+class FirstTVLiveIE(InfoExtractor):
+    IE_NAME = '1tv-live'
+    IE_DESC = 'Первый канал (прямой эфир)'
+    _VALID_URL = r'https?://(?:www\.)?1tv\.ru/live'
+    _GEO_BYPASS = False
+    _GEO_COUNTRIES = ['RU']
+
+    _TESTS = [
+        {
+            'url': 'https://www.1tv.ru/live',
+            'info_dict': {
+                'id': 'live',
+                'ext': 'mp4',
+                'title': 'ПЕРВЫЙ КАНАЛ ПРЯМОЙ ЭФИР СМОТРЕТЬ ОНЛАЙН',
+                'is_live': True,
+            },
+        },
+    ]
+
+    def _real_extract(self, url):
+        display_id = 'live'
+        webpage = self._download_webpage(url, display_id)
```

Comment:
Probably doesn't make a huge difference, but
```suggestion
        webpage = self._download_webpage(url, display_id, fatal=False)
```